files/templates/mbta/login/login-update-profile.ftl

```diff
@@ -109,18 +115,42 @@
 					/>
 	
 					<#if messagesPerField.existsError('lastName')>
-						<span id="input-error-lastname" class="${properties.kcInputErrorMessageClass!}" aria-live="polite">
-							${kcSanitize(messagesPerField.get('lastName'))?no_esc}
-						</span>
+						<div class="instructions-container">
+							<span id="input-error-lastname" class="${properties.kcInputErrorMessageClass!}" aria-live="polite">
+								${kcSanitize(messagesPerField.get('lastName'))?no_esc}
+							</span>
+						</div>
+					</#if>
+	            </div>
+	            
+	            <div class="form-group">
+	            	<label for="user.attributes.phone" class="form-input-label<#if messagesPerField.existsError('user.attributes.phone_number')> label-error</#if>">${msg("user.attributes.phone")}  <span>${msg("user.attributes.phone.span")}</label>
```

Comment:
suggestion: it looks like there are some other locations which need to be updated from `user.attributes.phone` to `user.attributes.phone_number` as well.